MC3-BeyondThe3F/MC3-BeyondThe3F/View/Components/AlbumCoverButtonComponentView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AlbumCoverButtonComponentView.swift
+//  MC3-BeyondThe3F
+//
+//  Created by 한영균 on 2023/07/19.
+//
+
+import SwiftUI
+
+struct AlbumCoverButtonComponentView: View {
+    var backgroundColor: CustomColor = .gray700
+    var foregroundColor: CustomColor = .white
```

Comment:
```suggestion
    private var backgroundColor: CustomColor = .gray700
    private var foregroundColor: CustomColor = .white
```

이 struct 외부에서 접근하는 경우가 없다면 접근제한자를 설정하면 어떨까요 ?
캡슐화로 안전하게 값을 보장할 수 있을것이라 생각해요 : )

Reply:
> 

접근제한자 모두 설정해두겠습니다!:)